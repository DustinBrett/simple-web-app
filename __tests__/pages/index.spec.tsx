import { render } from "@testing-library/react";
import Index from "pages/index";

test("renders main role", () => {
  const { container } = render(<Index />);

  expect(container).toMatchSnapshot();
});
