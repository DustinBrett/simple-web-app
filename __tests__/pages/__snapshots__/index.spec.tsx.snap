// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders main role 1`] = `
<div>
  <div
    class="ErrorMessage__StyledErrorMessage-sc-wuzvpj-0 gctktl"
  >
    Failed to load users
  </div>
</div>
`;
